airbyte-integrations/connectors/source-kafka/src/main/resources/spec.json

```diff
@@ -11,6 +11,50 @@
     "required": ["bootstrap_servers", "subscription", "protocol"],
     "additionalProperties": false,
     "properties": {
+      "MessageFormat": {
+        "title": "MessageFormat",
+        "type": "object",
+        "description": "The serialization used based on this ",
+        "oneOf": [
+          {
+            "title": "JSON",
+            "properties": {
+              "deserialization_type": {
+                "type": "string",
+                "enum": ["JSON"],
+                "default": "JSON"
+              }
+            }
+          },
+          {
+            "title": "AVRO",
+            "properties": {
+              "deserialization_type": {
+                "type": "string",
+                "enum": ["AVRO"],
+                "default": "AVRO"
+              },
+              "deserialization_strategy": {
+                "type": "string",
+                "enum": ["TopicNameStrategy","RecordNameStrategy","TopicRecordNameStrategy"],
+                "default": "TopicNameStrategy"
+              },
+              "schema_registry_url": {
```

Comment:
@sivankumar86 we need to add notes that Schema Registry is only supported on the AVRO format not JSON?

Reply:
Please add to `docs/integrations/sources/kafka.md`